src/components/network/network-table.js

```diff
@@ -50,10 +71,74 @@ genotet.NetworkTable.prototype.create = function(table, edges) {
       {title: 'Weight', data: 'weight'},
       {title: '', data: 'added'}
     ],
+    select: true,
+    dom: '<"row"<"col-sm-12"B>>' +
+      '<"row"<"col-sm-5"l><"col-sm-7"f>>' +
+      '<"row"<"col-sm-12"tr>>' +
+      '<"row"<"col-sm-5"i><"col-sm-7"p>>',
+    buttons: [
```

Comment:
yes, since the table supports selecting multiple edges.

Bowen

On Tuesday, January 26, 2016, Jiaming Dong <notifications@github.com> wrote:

> In src/components/network/network-table.js
> <https://github.com/ViDA-NYU/genotet/pull/75#discussion_r50892731>:
>
> > @@ -50,10 +71,74 @@ genotet.NetworkTable.prototype.create = function(table, edges) {
> >        {title: 'Weight', data: 'weight'},
> >        {title: '', data: 'added'}
> >      ],
> > +    select: true,
> > +    dom: '<"row"<"col-sm-12"B>>' +
> > +      '<"row"<"col-sm-5"l><"col-sm-7"f>>' +
> > +      '<"row"<"col-sm-12"tr>>' +
> > +      '<"row"<"col-sm-5"i><"col-sm-7"p>>',
> > +    buttons: [
>
> Yes at this time I only enable single edge addition/removal. Should I
> change it to multiple addition/removal?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/genotet/pull/75/files#r50892731>.
>